src/Tools/dotnet-monitor/Strings.resx

```diff
@@ -942,4 +942,11 @@
   <data name="ParameterCapturingNotAvailable_Reason_HotReload" xml:space="preserve">
     <value>The process has Hot Reload enabled.</value>
   </data>
+  <data name="ErrorMessage_ExpirationMustBePositive" xml:space="preserve">
+    <value>The expiration offset must be a positive time span value.</value>
+  </data>
+  <data name="HelpDescription_Expiration" xml:space="preserve">
+    <value>The expiration time on or after the generated key will no longer be accepted. This is a time span offset (e.g. "1:00:00" for one-hour) that will be added to the current date time to create the expiration date time.</value>
```

Comment:
nit: can you produce an example that uses days? I expect that to be the preferred amount.